tasks/42_first-n-smallest/first-n-smallest.js

```diff
@@ -0,0 +1,15 @@
+export function firstNSmallest(arr, numToReturn) {
+  if (numToReturn > arr.length) {
+    return [];
+  }
+  let threeSortedElements = arr
+      .slice()
+      .sort((a, b) => {
+        return a - b;
+      })
+      .slice(0, numToReturn);
+  return arr
+      .filter((item) => threeSortedElements.includes(item))
+      .slice(0, numToReturn);
```

Comment:
there is probably no reason for slice here as anyway you will have elements which in first array which already have needed length

Reply:
what about complexity of this solution? you do a lot of loops

Reply:
Agree, there is probably a better solution with fewer loops